### PR TITLE
Design improvements in pages

### PR DESCRIPTION
**Pull request by jajodiaraghav**
The commit handles the following:
- [x] Remove inline css
- [x] Remove useless JS snippets
- [x] Improve design
- [x] Refactor code for better readabilty
- [x] Fix indentation
- [x] Add a line at the end
- [x] Put subscription panels in a template

### CODE REVIEW

**Comment by jajodiaraghav** [comment]
@souravbadami Please review this PR.
It greatly improves the code quality. This will get us one step closer to adding a linter to our repository.

File: website/templates/email.html [hunk 1]
```diff
@@ -5,7 +5,9 @@
...
     </div>
-
-{{emails}}</br>
-
+    <div class="row">
+        <div class="col-lg-12">
+            <p>{{emails}}</p>
+        </div>
+    </div>
 {% endblock %}
```
Comment (souravbadami) [inline website/templates/email.html]: Add a line ending here.

File: website/templates/_subscriptions.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                <span class="pull-left" ><i class="fa fa-balance-scale" style="width:20px;"></i> $199 Bugheist / $1,300 tips</span></br>
+                <span class="pull-left" style="font-size:18pt;"><i class="fa fa-bullhorn"  style="width:25px;"></i> Weekly Announcement</span>
+                <span class="pull-left" style="font-size:20pt;"><i class="fa fa-globe"  style="width:25px;"></i> Homepage Featured</span>
+                <span class="pull-left" style="font-size:23pt;"><i class="fa fa-trophy"  style="width:25px;"></i> Monthly Hackathon</span>
+                <div class="clearfix"></div></br><center>
+                <button type="submit"  onclick="window.location.href='/hunt';" class="btn btn-primary btn-lg">Start</button></center>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (souravbadami) [inline website/templates/_subscriptions.html]: Add a line ending here.

File: website/static/css/main.css [hunk 3]
```diff
@@ -1,30 +1,68 @@
...
+    width: 86px;
+    font-size:28px;
+    margin-top:3px;
+    margin-right:10px;
+    color:#dd4252;
+}
+#login p {
+    font-size: 13px;
+    padding-top: 20px;
+}
```
Comment (souravbadami) [inline website/static/css/main.css]: Add a line ending here.

File: website/static/css/main.css [hunk 4]
```diff
@@ -1,30 +1,68 @@
...
-    /* border: 1px solid #000; */
     background: #eee;
     border-radius: 50%;
     font-size: 30px;
     box-shadow: 0 1px 1px #dd4252;
 }
-
-
 .no-shadow{
 	box-shadow: 0 0 0 #000;
```
Comment (souravbadami) [inline website/static/css/main.css]: Replace tab with 4 spaces.

File: website/static/css/main.css [hunk 5]
```diff
@@ -1,30 +1,68 @@
 .fa{
 	color: #555;
```
Comment (souravbadami) [inline website/static/css/main.css]: Replace tab with 4 spaces. Also, look for indentation with 4 spaces in this entire file. I can see several places that has to be fixed.

**Review by souravbadami — CHANGES_REQUESTED** [review]

**Comment by jajodiaraghav** [comment]
@souravbadami Updated the commit

**Review by souravbadami — APPROVED** [review]
LGTM.

**Pull request merged**